eth2/beacon/fork_choice/lmd_ghost.py

```diff
@@ -21,6 +24,47 @@
 from eth2.beacon.typing import Epoch, Gwei, SigningRoot, Slot, Timestamp, ValidatorIndex
 from eth2.configs import CommitteeConfig, Eth2Config
 
+LMD_GHOST_SCORE_DATA_LENGTH = 2
+
+
+def score_block_by_root(root: SigningRoot) -> int:
+    return int.from_bytes(root, byteorder="big")
+
+
+class LMDGHOSTScore(BaseScore):
+    _score: Tuple[Gwei, int]
+
+    def __init__(self, score: Tuple[Gwei, int]) -> None:
+        self._score = score
+
+    def __lt__(self, other: "LMDGHOSTScore") -> bool:
+        return self._score < other._score
+
+    def __eq__(self, other: Any) -> bool:
+        if not isinstance(other, LMDGHOSTScore):
+            return NotImplemented
+        return self._score == other._score
+
+    def serialize(self) -> bytes:
+        return ssz.encode(
+            self._score,
+            sedes=ssz.sedes.Vector(ssz.sedes.uint256, LMD_GHOST_SCORE_DATA_LENGTH),
+        )
+
+    @classmethod
+    def deserialize(cls, data: bytes) -> "LMDGHOSTScore":
+        score = ssz.decode(
+            data, sedes=ssz.sedes.Vector(ssz.sedes.uint256, LMD_GHOST_SCORE_DATA_LENGTH)
+        )
+        return cls(score)
+
+    @classmethod
+    def from_genesis(
+        cls, genesis_state: BeaconState, genesis_block: BaseBeaconBlock
+    ) -> BaseScore:
+        score = (Gwei(0), score_block_by_root(SigningRoot(ZERO_HASH32)))
+        return LMDGHOSTScore(score)
```

Comment:
nitpick, just notice that you use `return cls(score)` in `deserialize`.
```suggestion
        return cls(score)
```

Reply:
oh thank you! silly oversight on my part :)